### DISCUSSION TITLE
After update to v3.6.8 i'm unable to start uvtools

### DISCUSSION THREAD

**Discussion by frozenowni** [sn4k3/UVtools#574] [Q&A]
Hi,
after auto updating to version 3.6.8 (i think i had 3.6.1 before) i get the following error on startup:
"UVtools v3.6.8 X64 [Windows]
Unable to run due one or more missing dependencies.
Triggered by: libcvextern  (OpenCV)"

"Your system doesn't have the required dependencies in order to run.
Those dependencies are required at libcvextern/OpenCV library.
UVtools is built on top of the OpenCV and therefore cannot run.

Please install or build the dependencies in order to run the software.
Check the manual page at 'Requirements' section for help."

Has that happend to anyone else?

**Comment by sn4k3** [comment]
Happening to everyone, reinstall UVtools or repair. It will work

**Reply by Loneman-UK** [reply]
Is there any further information on this?  I've looked around, but this is the only thread to mention the issue.  

I've just allowed my install (a recent version) to auto-update, and it produced the issue frozenowni had.  
Uninstalling, reinstalling, and repairing have no effect, it merely displays the following: 
"UVtools v3.7.1 X64 [Windows]
Unable to run due one or more missing dependencies.
Triggered by: libcvextern  (OpenCV)"

Any help would be much appreciated.

**Reply by sn4k3** [reply]
Thats strange, i just upgraded on my laptop and it open fine.
Do the following:

1. Uninstall UVtools
2. Reboot your system
3. Redownload the msi
4. Install

Does it works?

**Reply by Loneman-UK** [reply]
Hi, thanks for the reply.
Nothing has worked so far, neither .msi installs nor the .zips, reboots included.

I'm going back through previous releases to see if I can find one that will launch.

**Comment by Loneman-UK** [comment]
Okay, so after downloading, extracting, running UVtools, closing once successfully launched and then deleting, starting with UVtools_win-x64_v3.5.0.zip and repeating the same all the way through to version 3.6.6, attempting to run v3.6.7 resulted in another non-starter.

  That's when I spotted the following in the changelog: *_"(Downgrade) OpenCV from 4.5.5 to 4.5.4 due a possible crash while detecting islands (Windows)",_* and that's the issue, for me at least.

  Using _any_ of the .zips after v3.6.6 means the programme won't start.  However, deleting the older "opencv_videoio_ffmpeg454_64.dll" from a newer zip and replacing it with the newer "opencv_videoio_ffmpeg455_64.dll" from an older, pre-v3.6.7 zip (confuzzled yet?) gets me a working UVTools with _all_ of the more recent versions.

  Whether I crash due to the island detection issue is yet to be seen.  Here's hoping this helps anyone else with the same problem.

**Reply by sn4k3** [reply]
That's very strange. Something isn't right there. I just tested on another fresh laptop and it works out of the box.
Can you printscreen the about box of that mix and post here?

**Reply by Loneman-UK** [reply]
Sure thing.
![UVT_Issue](https://user-images.githubusercontent.com/42150266/196823143-849c9ed1-30e5-4fff-a8eb-051bce1c1c99.png)

**Reply by sn4k3** [reply]
Need the about box print screen: Menu -> Help -> About

**Reply by Loneman-UK** [reply]
Here you go:

![UVt_About](https://user-images.githubusercontent.com/42150266/197059311-526a69e9-a85e-478b-8c9e-7f6b4341b2b2.png)

**Reply by sn4k3** [reply]
The opencv version is correct, i have no clue why it works with ffpmeg 4.5.5 and not 4.5.4, it makes no sense.
